modules/seqexec/web/client/src/main/scala/seqexec/web/client/handlers/SequenceExecutionHandler.scala

```diff
@@ -25,25 +25,31 @@ class SequenceExecutionHandler[M](
     with Handlers[M, SequencesQueue[SequenceView]] {
   def handleUpdateObserver: PartialFunction[Any, ActionResult[M]] = {
     case UpdateObserver(sequenceId, name) =>
-      val updateObserverE = Effect(SeqexecWebClient.setObserver(sequenceId, name.value).map(_ => NoAction))
-      val updatedSequences = value.copy(sessionQueue = value.sessionQueue.collect {
-        case s if s.id === sequenceId =>
-          s.copy(metadata = s.metadata.copy(observer = Some(name)))
-        case s                        => s
-      })
+      val updateObserverE = Effect(
+        SeqexecWebClient.setObserver(sequenceId, name.value).as(NoAction))
+      val updatedSequences =
+        value.copy(sessionQueue = value.sessionQueue.collect {
+          case s if s.id === sequenceId =>
+            s.copy(metadata = s.metadata.copy(observer = Some(name)))
+          case s => s
```

Comment:
Not a big deal, but it looks like the spacing alignment was thrown off by your IDE.